frontend/src/scenes/onboarding/onboardingLogic.tsx

```diff
@@ -212,6 +213,13 @@ export const onboardingLogic = kea<onboardingLogicType>([
                 return !product?.subscribed || !hasAllAddons || subscribedDuringOnboarding
             },
         ],
+        shouldShowReverseProxyStep: [
+            (s) => [s.product],
+            (product: BillingProductV2Type | null) => {
+                const productsWithReverseProxy = [ProductKey.FEATURE_FLAGS]
+                return productsWithReverseProxy.includes(product?.type as ProductKey)
```

Comment:
Let's flag this so we can run it as an experiment, I want to make sure this doesn't scare people away.